.github/workflows/regenerate_rubocop_todos.yml

```
@@ -0,0 +1,32 @@
+name: Regenerate Rubocop TODOs
+on:
+  push:
+    branches:
+      - depfu/update/srcapi/rubocop-*
+permissions:
+  contents: write
+jobs:
+  regenerate-rubocop-todos:
+    runs-on: ubuntu-latest
+    container: registry.opensuse.org/obs/server/unstable/containers/containers/openbuildservice/frontend-features:latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v4
+        with:
+          workflow: ${{ github.event.workflow_run.workflow_id }}
+          workflow_conclusion: success
+      - name: Install dependencies
+        run: |
+          cd src/api
+          bundler install
+      - name: Run rubocop
+        run: |
+          cd src/api
+          bundler exec rubocop --regenerate-todo
```

```suggestion
          bin/rake dev:lint:rubocop:auto_gen_config:rails
          bin/rake dev:lint:rubocop:auto_gen_config:root
```

I would use our dev task. And also regenerate the TODO file for the root subdirectory.